lib/block/transaction.go

```diff
@@ -79,10 +80,8 @@ func (bt BlockTransaction) NewBlockTransactionKeySource() string {
 
 func (bt BlockTransaction) NewBlockTransactionKeyConfirmed() string {
 	return fmt.Sprintf(
-		"%s%s%s%s",
+		"%s%s",
 		GetBlockTransactionKeyPrefixConfirmed(bt.Confirmed),
-		common.EncodeUint64ToByteSlice(bt.blockHeight),
```

Comment:
Do you think the confirmed time is enough to order block transactions? @spikeekips

Reply:
@kfangw Yes, I think so. The following `common.GetUniqueIDFromUUID()` supports to make sequential unique string in same machine, so `NewBlockTransactionKeyConfirmed()` will be sorted by creation.